backend/apiConfig.ts

```diff
@@ -0,0 +1,31 @@
+import 'dotenv/config'
+
+const production = process.env.NODE_ENV === 'production'
+type EnvOptions = { requireInProduction: boolean }
+const requiredInProduction: EnvOptions = { requireInProduction: true }
```

Comment:
Can this be removed? Only the notRequired... version appears to be used.

Reply:
that is true - I left it in incase we need it - but YAGNI  - I'll remove